src/dune_rules/merlin/merlin.ml

```diff
@@ -327,7 +327,15 @@ module Processed = struct
         |> Pp.concat_map ~sep:Pp.cut ~f:pp_one
         |> Pp.vbox
       in
-      Format.printf "%a%a@." Format.pp_set_margin 1000 Pp.to_fmt pp
+      let pp_format =
+        match Lazy.force Ansi_color.skip_line_break with
```

Comment:
Don't think there's much point to this configuration. Isn't this function only used for tests? Might as well just make it produce the output the tests want unconditionally.